docs/closure-compiler.md

```diff
@@ -379,14 +396,14 @@ export default class MDCExampleAdapter {
   /**
    * Registers an event listener `handler` for event type `type` on the root element.
    * @param {string} type
-   * @param {!Function} handler
+   * @param {function(!Event): undefined} handler
```

Comment:
Nit/question: Is there a preference RE `undefined` vs. `void` in closure type annotations?

Reply:
I don't know for sure, but I assume undefined is preferred. I have never used void.